express/blocks/bubble-ui-button/bubble-ui-button.js

```diff
@@ -0,0 +1,451 @@
+      showScrollArrow($wrapper, $lottie);
+    }
+    $wrapper.classList.remove('toolbox-opened');
+    if (userInitiated) {
+      setTimeout(() => {
+        if (!$wrapper.classList.contains('toolbox-opened')) {
+          $toolbox.classList.add('hidden');
+          $wrapper.classList.remove('clamped');
+          $button.classList.remove('toolbox-opened');
+        }
+      }, 500);
+    } else {
+      setTimeout(() => {
+        if ($wrapper.classList.contains('initial-load')) {
+          $toolbox.classList.add('hidden');
+          $wrapper.classList.remove('clamped');
+          $button.classList.remove('toolbox-opened');
+        }
+      }, 2000);
+    }
+    if (body) body.style.removeProperty('overflow');
+  } else {
+    $toolbox.classList.remove('hidden');
+    $wrapper.classList.add('clamped');
+    $button.classList.add('toolbox-opened');
+    hideScrollArrow($wrapper, $lottie);
+
+    setTimeout(() => {
+      $wrapper.classList.add('toolbox-opened');
+    }, 10);
+
+    setTimeout(() => {
+      bubbleUI.resizeBubbles($wrapper.querySelector('.bubble-ui'));
+    }, 500);
+    if (body) body.style.overflow = 'hidden';
+  }
+}
+
+function buildHexagon(values) {
+  // Create an empty result array
+  const result = [];
+
+  // Determine the dimensions of the honeycomb grid
+  // based on the number of values provided
+  const size = Math.ceil(Math.sqrt(values.length));
+  const half = Math.ceil(size / 2);
+
+  result.push(size);
+
+  for (let i = size - 1; i >= half; i -= 1) {
+    result.push(i);
+    result.unshift(i);
+  }
+
+  const accountedFor = result.reduce((partialSum, a) => partialSum + a, 0);
+
+  let counter = 0;
+  for (let i = accountedFor; i < values.length; i += 1) {
+    result[counter] += 1;
+
+    if (counter + 1 >= result.length) {
+      counter = 0;
+    } else {
+      counter += 1;
+    }
+  }
+
+  // Return the result array
+  return result;
+}
+
+function initBubbleUI(boxBottom) {
+  bubbleUI.centerBubbles(boxBottom);
+  bubbleUI.addEventListeners(boxBottom);
+
+  const vp = boxBottom.querySelector('.bubble-viewport');
+  if (vp) {
+    setTimeout(() => {
+      vp.addEventListener('scroll', () => {
+        const parent = boxBottom.closest('.floating-button-wrapper');
+        if (parent) {
+          parent.classList.remove('initial-load');
+        }
+      }, { passive: true });
+    }, 100);
+  }
+}
+
+async function decorateBubbleUI($boxBottom, data) {
+  $boxBottom.parentElement.classList.add('initial-load');
+  $boxBottom.classList.add('bubble-ui');
+  $boxBottom.classList.add('no-transition');
+
+  const bubbleViewportContainer = createTag('div', { class: 'bubble-viewport-container' });
+  const bubbleViewport = createTag('div', { class: 'bubble-viewport' });
+  const bubbleRowContainer = createTag('div', { class: 'bubble-row-container' });
+  const bubbleLoadBackground = createTag('div', { class: 'bubble-load-background' });
+
+  bubbleLoadBackground.append(getIconElement('aex-logo'));
```

Comment:
We already have the AX logo in `adobe-express`(.svg), do we really need another one?

Reply:
I'll actually remove this element for now... This logo here actually is visually different from `adobe-express.svg`. It's for a loading sequence that's currently on hold.